pyls/workspace.py

```diff
@@ -213,16 +213,9 @@ def word_at_position(self, position):
         return m_start[0] + m_end[-1]
 
     def jedi_names(self, all_scopes=False, definitions=True, references=False):
-        environment_path = None
```

Comment:
Why was all this code removed?

Reply:
This was redundant code (see line 229/222). `jedi.Script(...).get_names()` uses the environment specified in `jedi.Script(environment=...)`.

Reply:
Thanks!